src/components/FullStatistics.js

```diff
@@ -40,11 +40,7 @@ class FullStatistics extends Component {
               ? alarm.warning_threshold.warning_level
               : null}
           </td>
-          <td>
-            {alarm.warning_value
-              ? alarm.warning_value
-              : null}
-          </td>
+          <td>{alarm.warning_value ? alarm.warning_value : null}</td>
```

Comment:
I like this notation, but Dirk would rewrite it to `{ alarm.warning_value || null }` I think :)

Reply:
It was already there, this was prettifier doing that (I think). It's actually wrong here in both case since 0 is a valid warning_value that should be shown.